inc.php

```diff
@@ -33,12 +33,14 @@
 mb_internal_encoding('UTF-8');
 
 # for details see: http://de.php.net/manual/en/security.magicquotes.disabling.php
-if (get_magic_quotes_gpc()) {
-	$_POST = array_map('stripslashes_deep', $_POST);
-	$_GET = array_map('stripslashes_deep', $_GET);
-	$_COOKIE = array_map('stripslashes_deep', $_COOKIE);
-}
-
+#################################################################
```

Comment:
If the code is meaningless since PHP 5.4 and broken since PHP 8.0 you can simply remove it. :-)